class-tgm-plugin-activation.php

```diff
@@ -3108,6 +3108,7 @@ public function run( $options ) {
 					 *
 					 * @see Plugin_Upgrader::bulk_upgrade()
 					 * @see https://core.trac.wordpress.org/browser/tags/4.2.1/src/wp-admin/includes/class-wp-upgrader.php#L838
+					 * Last synced: Dec 31st 2015 against https://core.trac.wordpress.org/browser/trunk?rev=36134
```

Comment:
I'll adjust. All the same: the `{}` annotation is a quite recent change and hasn't been in use that long. Secondly the PSR is still in concept phase, so subject to change and as the annotation is a change, it may well change back to how it always used to be used, i.e. without the `{}`.